js/adapt-contrib-xapi.js

```diff
@@ -688,6 +688,11 @@ define([
         return;
       }
 
+      if (!model.get('_isComplete')) {
+        // The item is not actually completed, e.g. it may have been reset.
+        return;
+      }
```

Comment:
Seems this would resolve the issue but may have been simpler to actually pass in the second parameter to this method above i.e. `onItemComplete: function(model, isComplete) {` and then test for this immediately to save running all the other checks which would be redundant if `isComplete === false`